app-core.js

```diff
@@ -65,42 +65,13 @@ async function ensureSystemSettingsInitiated() {
 
   // Setup Settings as connection is established
   const serverSettings = await ServerSettings.init();
-  await logger.info(serverSettings);
-}
-
-async function serveOctoFarmNormally(app, port) {
-  if (!port || Number.isNaN(parseInt(port))) {
-    throw new Error(
-      "The server database-issue mode requires a numeric port input argument"
-    );
-  }
-
-  let listenerHttpServer = null;
-
-  logger.info("Initialising FarmInformation...");
-  await PrinterClean.initFarmInformation();
-
-  logger.info("Initialising Client Settings...");
   await ClientSettings.init();
+  logger.info(serverSettings);
 
-  logger.info("Initialising OctoFarm State...");
-  const { Runner } = require("./server_src/runners/state.js");
-  const stateRunnerReport = await Runner.init();
-  logger.info("OctoFarm State returned", stateRunnerReport);
-
-  logger.info("Initialising SystemRunner...");
-  const sr = await SystemRunner.init();
-  logger.info("OctoFarm SystemRunner returned", sr);
-
-  await optionalInfluxDatabaseSetup();
-
-  listenerHttpServer = app.listen(port, "0.0.0.0", () => {
```

Comment:
I moved the listen part back to `app.js` as this was the cause for headaches for API tests.